lib/cisco_node_utils/version.rb

```diff
@@ -14,7 +14,7 @@
 
 # Container module for version number only.
 module CiscoNodeUtils
-  VERSION = '1.5.0'
+  VERSION = '1.5.1'
```

Comment:
We are getting ready to release the `1.6.0` version of the gem but you can just leave it as `1.5.0` for now.